_pages/profile/submit/submit-profile/finalize-tab.js

```diff
@@ -83,7 +84,7 @@ class _FinalizeTab extends React.Component {
                   <Paragraph>
                     People can try to notify you of problems in your submission
                     and save your deposit via your{" "}
-                    <Link href="https://ethmail.cc/">ethmail.cc</Link>. Make sure
+                    <Link href="https://ethermail.io/">ethermail.io</Link>. Make sure
                     to check it while submission is being processed.
                   </Paragraph>
                 </>
```

Comment:
Color of the name element should be white instead of black

Reply:
Will fix on a subsequent pull.

Reply:
Update: I will remove this and have a proper setup once Etheremail effectively deploys their PoH integration.